server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/SignOffAuditRound.java

```diff
@@ -208,35 +209,30 @@ public String endpointBody(final Request request,
               cdb.ballotsAudited()));
 
           if (cdb.allAuditsComplete()) {
-            LOGGER.debug
-              (String.format
-               ("[signoff: RISK_LIMIT_ACHIEVED for %s County: "
-                + " cdb.estimatedSamplesToAudit()=%d,"
-                + " cdb.auditedSampleCount()=%d,"
-                + " cdb.ballotsAudited()=%d]",
-                cdb.county().name(),
-                cdb.estimatedSamplesToAudit(),
-                cdb.auditedSampleCount(),
-                cdb.ballotsAudited()));
             my_event.set(RISK_LIMIT_ACHIEVED_EVENT);
-            cdb.endAudits();
-            audit_complete = true;
+            final List<ComparisonAudit> terminated = cdb.endSingleCountyAudits();
```

Comment:
In this case, we'd be terminating single county audits for opportunistic benefits only.